src/utils/pgwire/src/pg_message.rs

```diff
@@ -506,8 +506,14 @@ impl<'a> BeMessage<'a> {
             // +-----+-----------+-----------------+
             BeMessage::CommandComplete(cmd) => {
                 let rows_cnt = cmd.rows_cnt;
-                let stmt_type = cmd.stmt_type;
+                let mut stmt_type = cmd.stmt_type;
                 let mut tag = "".to_owned();
+                stmt_type = match stmt_type {
+                    StatementType::INSERT_RETURNING => StatementType::INSERT,
```

Comment:
Seems there's currently no way to retrieve the "affected rows" for `RETURNING`. 🤔 Does it matter now? If not, we may wait for https://github.com/risingwavelabs/risingwave/issues/2678 to be resolved.

Reply:
Don't understand, currently the terminal will print correct affected rows, e.g., 
```
dev=> insert into t1 values (2+2), (4),(2),(4),(7),(8) returning *;
 v1 
----
  4
  4
  2
  4
  7
  8
(6 rows)

INSERT 0 6
```